tests/embedding/test_app.py

```diff
@@ -292,12 +292,19 @@ def test_building_gallery(pytestconfig, run_subprocess, rootdir):
 
 
 @pytest.mark.embedding
-def test_rm_lockfile(embedded_app, tmp_path: pytest.TempPathFactory):
```

Comment:
why did you remove test_rm_lockfile?

Reply:
It was not intended, I think I was adding this above it